OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -301,9 +301,9 @@ protected function generateNonce()
     /**
      * Configure the option resolver
      *
-     * @param OptionsResolverInterface $resolver
+     * @param OptionsResolver $resolver
      */
-    protected function configureOptions(OptionsResolverInterface $resolver)
+    protected function configureOptions(OptionsResolver $resolver)
```

Comment:
Alright, test failed on Travis. Not on my local. That's weird despite of I think Travis is right.

Will move this BC fix on another PR.